frontend/lib/package.json

```diff
@@ -103,6 +103,7 @@
     "styletron-react": "^6.1.0",
     "typed-signals": "^2.5.0",
     "typescript": "^4.9.5",
+    "ua-parser-js": "^1.0.39",
```

Comment:
`ua-parser-js` has switched to `AGPLv3` licenses in v2 last year (see announcement [here](https://github.com/faisalman/ua-parser-js/issues/680)) which would not be compatible :( But `ua-parser-js` might  still the best choice, even though we will not be able to update to 2.x?

Reply:
well that is unfortunate... but yeah given usage numbers i feel like 1.x might still be the best option. sounds like in that announcement google chrome is moving away from user agent entirely 🤔